<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph_window</class>
 <widget class="QWidget" name="Graph_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="graph_box" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>651</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y axis</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>971</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="x_axis_menu"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X-axis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_graph_button">
      <property name="text">
       <string>Clear Graph</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="y_axis_menu" native="true">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>29</y>
     <width>311</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <zorder>label_2</zorder>
  <zorder>widget</zorder>
  <zorder>graph_box</zorder>
  <zorder>y_axis_menu</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
